test/scalacopts/BUILD

```diff
@@ -0,0 +1,21 @@
+load("//scala:scala.bzl", "scala_macro_library", "scala_test")
+
+scala_macro_library(
+    name = "A",
+    srcs = ["A.scala"],
+)
+
+scala_test(
+    name = "EchoMacroSettings",
+    srcs = ["EchoMacroSettings.scala"],
+    scalacopts = [
+        # The macro settings are used in the test itself.
+        "-Xmacro-settings:name=a,b,c,d",
```

Comment:
Can we move “d” to  another option?
Are two different “Xmacro-settings” options accumulative?
If not maybe add another one with “name2=A”?
My idea is to have the test show the combination of two top level options where one of them has commas

Reply:
converted to 
```
    scalacopts = [
        # The macro settings are used in the test itself.
        "-Xmacro-settings:name=Mike,location=US",
        "-Xmacro-settings:hobby=basketball,soccer",
    ],
```